server/runtime_status.go

```diff
@@ -36,6 +36,30 @@ func (s *Server) Status(ctx context.Context, req *types.StatusRequest) (*types.S
 			},
 		},
 	}
+
+	for name, runtime := range s.config.Runtimes {
+		makeRuntimeHandler := func(name string, rro, userns bool) *types.RuntimeHandler {
+			return &types.RuntimeHandler{
+				Name: name,
+				Features: &types.RuntimeHandlerFeatures{
+					RecursiveReadOnlyMounts: rro,
+					UserNamespaces:          userns,
+				},
+			}
+		}
+
+		rro := runtime.RuntimeSupportsMountFlag("rro")
```

Comment:
This shouldn't be set until CRI-O actually implements the feature

Reply:
critest (https://github.com/kubernetes-sigs/cri-tools/pull/1344) is failing due to this

Reply:
Temporary workaround using:

- https://github.com/cri-o/cri-o/pull/7828

Fixed via:

- https://github.com/cri-o/cri-o/pull/7962